modules/trusted-firmware-m/Kconfig

```
@@ -16,7 +16,10 @@
 if BUILD_WITH_TFM
 
 config TFM_IPC
-	default y if TFM_PROFILE_TYPE_MINIMAL
+	default y
+
+config TFM_ISOLATION_LEVEL
+	default 2
```

Still need the type:
```suggestion
config TFM_ISOLATION_LEVEL
        int
	default 2
```

---

It doesn't seem so, it works without the type. Probably because it firstly read the relevant Zephyr Kconfig and then just load this default value.